UKHomeOffice/sgmr-service
# Rename 'vessel' to 'pleasure craft'

## Description
We have new user flows being tested with users. In the meantime, we should take the content on the prototype and update the existing pages on the service to match.

All user-visible mentions of "vessel" have been replaced by “pleasure craft”.

## To Test
- Go to any page which would have previously contained "vessel"
- Ctrl + F in your browser and type "vessel"
- You should get no results
- Ctrl + F in your browser and type “pleasure craft”
- You should get at least one result

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
